src/prefect/cli/root.py

```diff
@@ -114,14 +113,9 @@ async def version():
         "Version": prefect.__version__,
         "API version": SERVER_API_VERSION,
         "Python version": platform.python_version(),
-        "Git commit": prefect.__version_info__["full-revisionid"][:8],
-        "Built": pendulum.parse(
-            prefect.__version_info__["date"]
-        ).to_day_datetime_string(),
```

Comment:
is this worth replacing? for published versions we could look this up, but I'm not sure its necessary